docs/source/topics/data_types.rst

```diff
@@ -1149,6 +1149,8 @@ Therefore, we have to override the constructor :meth:`~aiida.orm.nodes.node.Node
 .. warning::
 
     For the class to function properly, the signature of the constructor **cannot be changed** and the constructor of the parent class **has to be called**.
+    Note that the constructor is **NOT** called when the node is loaded from the database afterwards.
+    Hence, one should avoid setting attributes that correspond to the sorted data inside the ``__init__`` itself. Instead, use the ``@property`` approach to provide easy access to any sorted data instead (see below).
```

Comment:
Not sure what you the "sorted data" was referring to. Also, I think the wording should be slightly different since it is fine to set attributes (as in attributes stored in the database) in the constructor, it is just that one should not rely on setting "instance attributes" in the constructor as that is not called when loading. But maybe that is what you were also saying. Maybe a slightly clearer version could be:
```suggestion
    Note also that the constructor is **NOT** called when the node is loaded from the database afterwards.
    Hence, one should not rely on initializing instance attributes inside the ``__init__`` itself (here "attributes" does not refer to the data stored in the database, but the normal Python understanding of attributes that class instances have).
```